src/view/MenuView.java

```diff
@@ -46,6 +50,17 @@ public MenuView(MenuViewModel menuViewModel, ViewManagerModel viewManagerModel)
         loadGame.addActionListener(this);
         careerStats.addActionListener(this);
 
+        this.careerStats.addActionListener(new ActionListener() {
+            @Override
+            public void actionPerformed(ActionEvent e) {
+                if (e.getSource().equals(MenuView.this.careerStats)) {
+                    StatisticsState statisticsState = statisticsViewModel.getState();
+                    statisticsController.execute();
+                    JOptionPane.showMessageDialog((Component)null, statisticsState.stats);
+                }
+            }
```

Comment:
Im not sure this is good practice according to CA.  I think the message dialog should be called when firePropertyChnage is triggered by the presenter. That way you also solve the problem having the `parentObject` be `null`.